Content.Server/Health/BodySystem/BodyManagerComponent.cs

```diff
@@ -191,10 +191,10 @@ public List<BodyPart> GetBodyPartsOfType(BodyPartType type)
         /// <summary>
         ///     Disconnects the given BodyPart reference, potentially dropping other BodyParts if they were hanging off it. 
```

Comment:
Please add a note here that it returns either null or the dropped entity, if it spawns one.

Reply:
You should do that inside a `<returns>` tag.